src/e2e-test/features/BQExecuter/BQExecuteErrorScenarios.feature

```diff
@@ -0,0 +1,44 @@
+@BQExecute
+Feature: Verify BigQueryExecute plugin error scenarios
+
+  @BQExecute
+  Scenario: Verify BigQueryExecute validation error for mandatory field SQL Query
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Validate mandatory property error for "sql"
+
+  @BQExecute
+  Scenario: Verify BigQueryExecute validation error for Blank Table name
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQueryExecute property sql "bqExecuteQuery"
+    Then Enter BigQueryExecute property dataset "dataset"
+    Then Verify the BigQuery validation error message for invalid dataset tablename property "dataset"
+
+  @BQExecute
+  Scenario: Verify BigQueryExecute Query with Blank Dataset and Table name
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQueryExecute property sql "bqExecuteQuery"
+    Then Enter BigQueryExecute source property table name
+    Then Verify the BigQuery validation error message for invalid dataset property "dataset"
+
+  @BQExecute
+  Scenario: Verify Replace BigQueryExecute property Job Location with blank value
```

Comment:
-> Verify BigQueryExecute validation error for mandatory field job location

Reply:
@sawantpritam -done